src/components/Header/index.jsx

```diff
@@ -40,6 +40,7 @@ class Header extends PureComponent {
 
   render() {
     const { signoutDropDownIsVisible } = this.state;
+    console.log(this.props)
```

Comment:
I think you should remove this console log command

Reply:
Good eye. Thanks